### PR TITLE
Fix#12 UI Fixes

### PR DESCRIPTION
**Pull request by zephyr004**
## Overview

1. This PR fixes or fixes part of #12.
2. This PR does the following: Made UI changes in Login Activity and Sign Up Activity maintaining consistency, following the Figma design and the required guidelines.

## Essential Checklist

- [✅ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [✅ ] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
- [ ✅] The PR is made from a branch that's **not** called "main/master".

## Proof that changes are correct
Screenshot 2024-06-13 231940

## PR Pointers

- If you need a review or an answer to a question, and don't have permissions to assign people, **leave a comment** like the following: "{{Question/comment}} @{{reviewer_username}} PTAL".
- Never force push. If you do, your PR will be closed.

### CODE REVIEW

**Comment by Agrim2112** [comment]
@zephyr004 Increase the font sizes a bit and increase the top margin of the edit text fields in Login Activity. Also, have you tested it on physical devices?

**Comment by zephyr004** [comment]
@Agrim2112 Font size and the mentioned margin in Login Activity has been increased. Attached below is the image of how the layout looks on physical device.

Screenshot 2024-06-15 005459

File: app/src/main/res/layout/activity_sign_up.xml [hunk 1]
```diff
@@ -131,69 +149,79 @@
...
-                    android:textSize="20sp"
+                    android:textSize="24sp"
+                    app:layout_constraintBottom_toBottomOf="parent"
                     app:layout_constraintEnd_toEndOf="parent"
                     app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toBottomOf="@+id/etPassword" />
+                    app:layout_constraintTop_toBottomOf="@+id/etPassword"
+                    app:layout_constraintVertical_bias="0.651" />
 
                 <TextView
```
Comment (Agrim2112) [inline app/src/main/res/layout/activity_sign_up.xml:195]: Wrap these text views in a Linear Layout and constrain the Linear Layout in the Constraint Layout accordingly.

File: app/src/main/res/layout/activity_sign_up.xml [hunk 2]
```diff
@@ -56,47 +65,51 @@
...
                     android:id="@+id/etName"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
+                    android:layout_marginHorizontal="36dp"
                     android:layout_marginTop="12dp"
-                    android:layout_marginHorizontal="50dp"
-                    android:padding="5dp"
                     android:background="@drawable/custom_edittext"
-                    android:elevation="5dp"
+                    android:elevation="4dp"
```
Comment (Agrim2112) [inline app/src/main/res/layout/activity_sign_up.xml:107]: Reduce the elevation to 1dp for all Edit Text Views

File: app/src/main/res/layout/activity_sign_up.xml [hunk 3]
```diff
@@ -56,47 +65,51 @@
...
+                    android:hint="@string/hint_name"
+                    android:fontFamily="@font/inter_regular"
+                    android:padding="4dp"
```
Comment (Agrim2112) [inline app/src/main/res/layout/activity_sign_up.xml:110]: Increase the padding to 12dp for all Edit Text Views. Ensure that the layout fits in the screen, if it doesn't reduce padding accordingly

**Review by Agrim2112 — CHANGES_REQUESTED** [review]

**Comment by Agrim2112** [comment]
@zephyr004 Looks good, just need to make some minute changes. Also, ensure that the button signup/login button in both the activities are positioned similarly.

File: app/src/main/res/layout/activity_login.xml [hunk 4]
```diff
@@ -193,31 +193,44 @@
...
-                    android:text="@string/sign_up"
-                    android:textColor="@color/btn_color"
-                    android:textIsSelectable="true"
-                    android:textSize="16sp"
-                    app:layout_constraintStart_toEndOf="@+id/tvDontHaveAccount"
-                    app:layout_constraintTop_toTopOf="@+id/tvDontHaveAccount" />
-
+                <androidx.constraintlayout.widget.ConstraintLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent">
```
Comment (Agrim2112) [inline app/src/main/res/layout/activity_login.xml:198]: Remove this Constraint Layout, use the root Constraint layout to position the Linear Layout

**Review by Agrim2112 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_login.xml (hunk 4 above)
Comment (zephyr004) [reply app/src/main/res/layout/activity_login.xml]: I have removed the additional layout as suggested.

**Comment by Agrim2112** [comment]
@zephyr004 Good work

**Pull request merged**